aurora/radiation.py

```diff
@@ -686,17 +685,23 @@ def read_adf15(path, order=1):
         # Get the wavelength, number of densities and number of temperatures
         # from the first line of the entry:
         l = lines.pop(0)
-
+ 
         header_dict = {}
 
-        header = l.split("/")
-        lam, sizes = header[0].split("A")
-        header_dict["lam"] = float(lam)
-        num_den, num_temp = np.int_(sizes.split())
+        header = l.split('/')
+
+        header0 = header[0].split()
+        header_dict['lam'] = float(header0[0].strip('A'))
+        num_den = np.int_(header0[-2])
+        num_temp = np.int_(header0[-1])
 
         for item in header[1:]:
-            par, val = item.split("=")
-            header_dict[par.strip().upper()] = val.strip()
+            try:
```

Comment:
@odstrcilt could you please explain why you needed to add this try-except? Please add a comment.